static/app/components/events/eventExtraData/eventExtraData.tsx

```diff
@@ -10,39 +10,21 @@ type Props = {
   event: Event;
 };
 
-type State = {
-  raw: boolean;
-};
-
-class EventExtraData extends Component<Props, State> {
-  state: State = {
-    raw: false,
-  };
-
-  shouldComponentUpdate(nextProps: Props, nextState: State) {
-    return this.props.event.id !== nextProps.event.id || this.state.raw !== nextState.raw;
-  }
-
-  toggleRaw = (shouldBeRaw: boolean) => {
-    this.setState({
-      raw: shouldBeRaw,
-    });
-  };
-
-  render() {
+const EventExtraData = memo(
+  ({event}: Props) => {
+    const [raw, setRaw] = useState(false);
     return (
-      <div className="extra-data">
```

Comment:
Was there any CSS attached to this that could be removed?

Reply:
there is a css, but it looks like it was not having any impact on the UI. I will remove the css too